docs/madr/1-intial.md

```diff
@@ -0,0 +1,97 @@
+# Asset resolution by way of an ANT Registry
+
+- Status: accepted
+- Deciders: [Dylan], [Ariel], [Phil]
+- Date: 2024-07-22
+- Authors: [Atticus], [Dylan], [Ariel], [Phil]
+
+## Context and Problem Statement
+
+Performance of loading a user’s registered name is slow with the current method of loading all of the ArNS ecosystem (all ArNS names and ANTs registered to them). There is no way to quickly identify ownership of an ANT, let alone any AO process.
+
+## Decision Drivers
+
+The main drivers for this decision revolve around:
+
+- **Speed**
+  - Uses fewer resources client-side.
+- **Scalability**
+  - We cannot load the entire ArNS ecosystem client-side if it grows.
+- **Ecosystem Integrations**
+  - Having this utility process solely for ANT contract indexing provides integration points for other applications and reduces our reliance on other teams/products (e.g., Bazar profile processes).
+- **Maintainability**
+  - Since we control the registry process, we can drive its maintenance.
+
+## Considered Options
+
+### ANT Registry
+
+- **Pros:**
+  - Dedicated registry for ANTs.
+  - Event-driven updates from ANTs.
+  - Separate state and compute from the resolver process.
```

Comment:
, thus preserving system concurrency while providing for better UX in the system.